prow/config.yaml

```diff
@@ -13743,6 +13743,29 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
+- interval: 24h
+  name: ci-kubernetes-e2e-kops-aws-kopeiovxlan
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170731-2e38747a
```

Comment:
you'll need to commit kubetest changes first, then build a new image from there :-)